cmake/Helpers.cmake

```diff
@@ -269,7 +269,13 @@ FUNCTION(CONFIGURE_SDL SDL_BACKEND)
         INCLUDE(FindSDL)
         IF(USE_SDLMIXER)
             FIND_PACKAGE(SDL_mixer MODULE)
+            SET_INTERNAL(SDL_MIXER_INCLUDE_DIR ${SDL2_MIXER_INCLUDE_DIRS})
+            SET_INTERNAL(SDL_MIXER_LIBRARIES ${SDL2_MIXER_LIBRARIES})
         ENDIF()
+
+        SET_INTERNAL(SDL_INCLUDE_DIR ${SDL_INCLUDE_DIRS})
+        SET_INTERNAL(SDL_LIBRARY "${SDL2_LIBRARIES}")
```

Comment:
Won't all these be empty? SDL1 does not set SDL2_MIXER_INCLUDE_DIRS etc.

Reply:
You're right. I noticed that. It didn't happened locally since I forgot to wipe out CMake cache ;)

Reply:
This has not been fixed.